game_event/templates/game_event/game-event.html

```diff
@@ -2,72 +2,89 @@
+    <div class="table-container">
         <table class="nice-table">
             <thead>
                 <tr>
-                <th> </th>
-                <th>Name</th>
-                <th>Brings Ball</th>
-            </tr>
+                    <th> </th>
+                    <th>Name</th>
+                    <th>Brings Ball</th>
+                </tr>
             </thead>
-                <tbody>
-                    {% for player in event_players %}
-                        <tr>
-                            <td>
-                                <img src="{% static 'images/favicon.ico' %}" alt="Ball Image" class="ball-image">
-                            </td>
-                            <td>{{ player.first_name }} {{ player.last_name }}</td>
-                            <td>
-                                {% if player.brings_ball == True %}
-                                    Yes
-                                {% else %}
-                                    No
-                                {% endif %}
-                            </td>
-                        </tr>
-                    {% endfor %}
-                </tbody>
-            </table>
-            <div>
-                {% if in_event %}
-                    <a href="/game-events/remove-from-event/{{ id }}/"
-                    class="round-button"
-                    onclick="window.open(this.href, 'Remove Event', 'width=400,height=250,left='+((screen.width-400)/2)+',top='+((screen.height-515)/2)); return false;">
-                    Remove Event
-                    </a>
-                {% else %}
-                    <a href="/game-events/join-event/{{ id }}/"
-                    class="round-button"
-                    onclick="window.open(this.href, 'Join Event', 'width=400,height=250,left='+((screen.width-400)/2)+',top='+((screen.height-515)/2)); return false;">
-                    Join Event
-                    </a>
-                {% endif %}
-            </div>
-       {% else %} 
-        <div>
-            <h1> Event Does not exist. </h1>
-        </div>
-       {% endif %}
-  </body>
+            <tbody>
+                {% for player in event_players %}
+                <tr>
+                    <td>
+                        <img src="{% static 'images/favicon.ico' %}" alt="Ball Image" class="ball-image">
+                    </td>
+                    <td>{{ player.first_name }} {{ player.last_name }}</td>
+                    <td>
+                        {% if player.brings_ball == True %}
+                        Yes
+                        {% else %}
+                        No
+                        {% endif %}
+                    </td>
+                </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    </div>
+    <div class="button-container">
+        {% if in_event %}
+        <a href="/game-events/remove-from-event/{{ id }}" class="round-button remove-event-button">
+            Remove Event
```

Comment:
Remove event sounds like the event is being removed and not the player-event
```suggestion
            Leave Event
```